package.json

```diff
@@ -16,7 +16,7 @@
     "postinstall": "husky install",
     "prepare": "node -e \"try { require('husky').install() } catch (e) {if (e.code !== 'MODULE_NOT_FOUND') throw e}\"",
     "test": "jest -c jest.config.ts",
-    "test:integration": "jest --setupFiles ./.integration.test-${ENV:-build}.env -c jest.integration.config.ts"
+    "test:integration": "jest -c jest.integration.config.ts -i"
```

Comment:
Please can we put all the test related stuff including config in a `tests` folder, with an sub-folder for their type i.e. `tests/integration-tests/...` or `tests/e2e-tests/...` basically can we keep consistency with the structure in the other repo.

Reply:
Done.